README.md

```diff
@@ -3,7 +3,7 @@
 Hackathon registration framework. Used in TreeHacks since 2019. See it in action at https://root.treehacks.com
 
 ## Local setup
-1. Make sure you have MongoDB installed (so `mongod` is in your PATH)
+1. Make sure you have MongoDB installed (so `mongod` is in your PATH), on MacOS make sure that the `data/db` directory exists.
```

Comment:
This is not specific to macOS; it's always required to do this. 

Also, `data/db` is in which folder?